cg_hermes/constants/tags.py

```diff
@@ -73,6 +74,12 @@ class VariantTags(StrEnum):
     VCF2CYTOSURE: str = "vcf2cytosure"
     VCF: str = "vcf"
     VCF_INDEX: str = "vcf-index"
+    VCF_MT: str = "vcf_mt"
+    VCF_MT_INDEX: str = "vcf_mt_index"
+    VCF_MT_CLINICAL: str = "clinical_mt"
+    VCF_MT_CLINICAL_INDEX: str = "clinical_mt_index"
+    VCF_MT_RESEARCH: str = "research_mt"
+    VCF_MT_RESEARCH_INDEX: str = "research_mt_index"
```

Comment:
all (or most) HK tags use `-`, maybe you can use the same for consistency

Reply:
Done